src/main/webapp/app/shared/modal/AddVusModal.tsx

```diff
@@ -54,12 +55,16 @@ const AddVusModalV2 = (props: IAddVusModalProps) => {
     }
   }, [variants, mutationList, props.vusList]);
 
+  function handleVariantAdded() {
+    const filteredAlterations = filterAlterationsAndNotify(inputValue);
+    setVariants(state => [...state, ...filteredAlterations.map(alt => createOption(alt))]);
+    setInputValue('');
+  }
+
   const handleKeyDown = event => {
     if (!inputValue) return;
     if (event.key === 'Enter' || event.key === 'tab') {
```

Comment:
I think we don't need to check for tab anymore?

Reply:
I think it's fine. I don't think there's any harm in allowing the tab key to behave like the enter key, I see online that some people expect that. Will change the language though as you mentioned.